lib/jsonnet/evaluator.rb

```diff
@@ -0,0 +1,46 @@
+module Jsonnet
+  class Evaluator
```

Comment:
Yup that looks good to me 👍 

I'm a bit confused about the need for: https://github.com/yugui/ruby-jsonnet/blob/4eca92cd7b87c2f1a1d6749266d8015f5d730448/lib/jsonnet/vm.rb#L9-L13 but that could be because I'm not too hot on the ruby c interface.